### PR TITLE
Add tests for guest user and logged in user viewing category drop-down

### PR DESCRIPTION
**Pull request by cmacaulay**
👍 
- All tests still passing

### CODE REVIEW

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb
Comment (meyerhoferc) [inline spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:34]: 👍

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb
Comment (meyerhoferc) [inline spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:53]: 👍

**Review by meyerhoferc — APPROVED** [review]
glad we have tests for these now, ready for second review & merge @CjMoore @valerietrudell

File: spec/features/nav_footer/user_can_navigate_site_using_nav_and_footer_spec.rb
Comment (CjMoore) [inline spec/features/nav_footer/user_can_navigate_site_using_nav_and_footer_spec.rb:2]: Super thorough tests. Thanks.

**Review by CjMoore — APPROVED** [review]
Looks great merging now!

**Pull request merged**
